client/apps/queue/routes.coffee

```diff
@@ -0,0 +1,28 @@
+_ = require 'underscore'
+query = require './query.coffee'
+Q = require 'bluebird-q'
+Lokka = require('lokka').Lokka
+Transport = require('lokka-transport-http').Transport
+{ API_URL } = process.env
+
+@queue = (req, res, next) ->
+  published = true
+  channel_id = req.user?.get('current_channel').id
+  queuedQuery = query "daily_email: true"
+  latestQuery = query "channel_id: \"#{channel_id}\", published: true, sort: \"-published_at\", daily_email: false"
+
+  client = new Lokka
+    transport: new Transport(API_URL + '/graphql')
+
+  client.query(queuedQuery).then (result) =>
+    if result.error
+      return res.backboneError
+    res.locals.sd.QUEUED_ARTICLES = result.articles
+    client.query(latestQuery).then (result) =>
```

Comment:
Nesting the queries like this is really not preferred and soo counterintuitive with GraphQL, but I couldn't get around using aliases with JoiQL simply. See: https://github.com/muraljs/joiql/issues/11